src/url/cloudinaryURL.ts

```diff
@@ -7,9 +9,60 @@ import Transformation from '../transformation/Transformation';
  * @param {Object} descriptor
  * @param {Transformation} transformation
  */
-function createCloudinaryURL(config: unknown, descriptor: unknown, transformation: unknown) {
-  // Currently not implemented
-  return '';
+function createCloudinaryURL(config: CloudinaryConfig, descriptor?: IDescriptor, transformation?: Transformation) {
```

Comment:
is that a public interface?

Reply:
In theory it could be, it's currently used through the transformableImage class

```  
toURL(): string {
    return createCloudinaryURL(this.config, Object.assign({
      resourceType: 'image',
      type: 'upload'
    }, this.asset), this);
  }
```